tests/set_get_delete_500.bash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $TRACE ]] && set -x
```

Comment:
Awesome yes!

Okay so I'll do it bit by bit:

### set -e

Exit immediately if a command exits with a non-zero status

### set -o

Used to set an option

### pipefail

If set, the return value of a pipeline is the value of the last (rightmost) command to exit with a non-zero status, or zero if all commands in the pipeline exit successfully. This option is disabled by default.

### `[[ $TRACE ]]`

Only do something if the `TRACE` environment variable exists and has a *any* non-zero length value.

### set -x

Prints a trace of each command run.

## All Together Now!

Exit immediately for any pipe failures, and enable enhanced tracing of what commands are actually being run (useful for debugging what a script is doing) when TRACE=1.